rskj-core/src/main/java/co/rsk/rest/modules/RestModuleLoader.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * This file is part of RskJ
+ * Copyright (C) 2018 RSK Labs Ltd.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+package co.rsk.rest.modules;
+
+import co.rsk.rest.dto.RestModuleConfigDTO;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+
+public class RestModuleLoader {
+
+    private final List<RestModule> restModuleList;
+
+    public RestModuleLoader(RestModuleConfigDTO restModuleConfigDTO) {
+        Objects.requireNonNull(restModuleConfigDTO, "REST Module Config can not be null");
+
+        restModuleList = new ArrayList<>();
+
+        restModuleList.add(new HealthCheckModule("/health-check",
```

Comment:
Here would be a nice place to do the filtering by enabled/disabled module. If it is not active then we wouldn't need it into the list. what do you think?

Reply:
In current implementation `Dispatcher` is in charge of checking module status (active/inactive). In the other hand, module quantity will be always a small, so I think we can have the complete list here, and check status when needed.

If, in the future, we observe some overhead that can be avoided, we can add some logic at `Dispatcher` level so it knows when to ignore some module beforehand.